contracts/extensions/ccd001-direct-execute.clar

```diff
@@ -20,7 +20,7 @@
 
 (define-constant ERR_UNAUTHORIZED (err u3000))
 (define-constant ERR_NOT_APPROVER (err u3001))
-(define-constant ERR_ALREADY_EXECUTED (err u3002))
+;; not used (define-constant ERR_ALREADY_EXECUTED (err u3002))
```

Comment:
good catch - if we're not using it then should remove, is it worth renumbering the errors in order?

Reply:
yes i wondered if this causes more confusion than just removing and leaving hole - i prefer not renumbering and leaving the hole.

Reply:
Normally I'd agree, but I think it makes sense to renumber this because it shouldn't have been there in the first place. Looking at other xDAO implementations with similar contracts:

- [ecosystemDAO](https://github.com/Clarity-Innovation-Lab/ecosystem-dao/blob/feat/baseline/contracts/extensions/ede003-emergency-proposals.clar#L21-L24)
- [stackerDAO](https://github.com/StackerDAOs/backend/blob/main/contracts/extensions/executive-team/sde-emergency-proposals.clar#L15-L18)

Looks like the original code here is where it came from:

- [executorDAO](https://github.com/MarvinJanssen/executor-dao/blob/main/contracts/extensions/ede004-emergency-execute.clar#L20-L24)